src/oc-id/app/assets/images/example/logo-tm.svg

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Are these files in the `examples/` folder required?

Reply:
@stevendanna I copied all the assets of `chef-web-core` repo to resolve `chef-web-core` dependency, because we have removed the `chef-web-core` gem.

I'll check if it can be remove.